src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in

```diff
@@ -86,18 +87,34 @@ class BasePredict():
         if self.pred_type == 1:
             rank_create_sql = ""
 
-        self.pred_vartype = self.dependent_vartype.strip('[]')
-        unnest_sql = ''
-        if self.pred_vartype in ['text', 'character varying', 'varchar']:
+        self.pred_vartype = [i.strip('[]') for i in self.dependent_vartype]
+        unnest_sql = []
+        full_class_name_list = []
+        full_class_value_list = []
+
+        for i in range(self.dependent_var_count):
+
+            if self.pred_vartype[i] in ['text', 'character varying', 'varchar']:
+
+                unnest_sql.append("unnest(ARRAY{0}) AS {1}".format(
+                    ['NULL' if j is None else j for j in class_values[i]],
+                    self.dependent_varname[i]))
```

Comment:
I think this should be:
   quote_ident(self.dependent_varname[i])
instead of:
   self.dependent_varname[i]